docs/src/drivers/mb2hal.adoc

```diff
@@ -103,56 +139,82 @@ Modbus transaction function code (see link:https://modbus.org/specs.php[specific
+* 0x01 - ILLEGAL_FUNCTION        - the FUNCTION code received in the
+  query is not allowed or invalid.
+* 0x02 - ILLEGAL_DATA_ADDRESS    - the DATA ADDRESS received in the
+                                   query is not an allowable address for
+                                   the slave or is invalid.
+* 0x03 - ILLEGAL_DATA_VALUE      - a VALUE contained in the data query
+                                   field is not an allowable value or is
+                                   invalid.
+* 0x04 - SLAVE_DEVICE_FAILURE    - SLAVE (or MASTER) device unrecoverable
+                                   FAILURE while attempting to perform
+                                   the requested action.
 * 0x04 - SERVER_FAILURE          - (see above).
-* 0x05 - ACKNOWLEDGE             - This response is returned to PREVENT A TIMEOUT in the master.
-                                   A long duration of time is required to process the request in the slave.
-* 0x06 - SLAVE_DEVICE_BUSY       - The slave (or server) is BUSY. Retrasmit the request later.
+* 0x05 - ACKNOWLEDGE             - This response is returned to
+                                   PREVENT A TIMEOUT in the master.
+                                   A long duration of time is required
+                                   to process the request in the slave.
+* 0x06 - SLAVE_DEVICE_BUSY       - The slave (or server) is BUSY.
+                                   Retransmit the request later.
 * 0x06 - SERVER_BUSY             - (see above).
-* 0x07 - NEGATIVE_ACKNOWLEDGE    - Unsuccessful programming request using function code 13 or 14.
+* 0x07 - NEGATIVE_ACKNOWLEDGE    - Unsuccessful programming request
+                                   using function code 13 or 14.
 * 0x08 - MEMORY_PARITY_ERROR     - SLAVE parity error in MEMORY.
-* 0x0A (-10) - GATEWAY_PROBLEM_PATH    - Gateway path(s) not available.
-* 0x0B (-11) - GATEWAY_PROBLEM_TARGET  - The target device failed to repond (generated by master, not slave).
+* 0x0A (-10) - GATEWAY_PROBLEM_PATH   - Gateway path(s) not available.
+* 0x0B (-11) - GATEWAY_PROBLEM_TARGET - The target device failed to
+                                        respond (generated by master,
+                                        not slave).
 
 Program or connection:
 
-* 0x0C (-12) - COMM_TIME_OUT           
-* 0x0D (-13) - PORT_SOCKET_FAILURE     
-* 0x0E (-14) - SELECT_FAILURE          
-* 0x0F (-15) - TOO_MANY_DATAS          
-* 0x10 (-16) - INVALID_CRC             
-* 0x11 (-17) - INVALID_EXCEPTION_CODE  
+* 0x0C (-12) - COMM_TIME_OUT
+* 0x0D (-13) - PORT_SOCKET_FAILURE
+* 0x0E (-14) - SELECT_FAILURE
+* 0x0F (-15) - TOO_MANY_DATAS
+* 0x10 (-16) - INVALID_CRC
+* 0x11 (-17) - INVALID_EXCEPTION_CODE
 
+////
 == Example config file
 
+FIXME Files missing
+
 Click link:mb2hal_HOWTO.ini[here] to download.
 
 [source,{ini}]
 ----
 include::mb2hal_HOWTO.ini[]
 ----
+////
```

Comment:
The file is copied by the Submakefile, so no problem here...

Reply:
FIXED thanks